### ISSUE TITLE
Syntax error pulling docker

### ISSUE THREAD

**Issue by locallycompact** [kylemanna/docker-aosp#18]
Hi, I'm getting the following error trying to use this docker in a project.

    [: -c: line 1: syntax error: unexpected end of file

https://gitlab.com/locallycompact/aosp/builds/8170460

This error is happening during the docker creation before it begins executing.

https://gitlab.com/locallycompact/aosp/blob/0da9cd6199d4b129a885a9a72f4d0c2ee337ab69/.gitlab-ci.yml

Thanks

**Comment by kylemanna** [comment]
Took a quick look and I don't see any problems on the image side.

    docker run --rm -it kylemanna/aosp echo "Hello World"
    docker_entrypoint: Creating user UID/GID [1000/1000]
    docker_entrypoint: Creating user UID/GID [1000/1000] - done
    docker_entrypoint: Copying .gitconfig and .ssh/config to new user home
    docker_entrypoint: Copying .gitconfig and .ssh/config to new user home - done
    docker_entrypoint: Creating /tmp/ccache and /aosp directory
    docker_entrypoint: Creating /tmp/ccache and /aosp directory - done

    Hello World

And the gitlab-ci looks right.  Furthermore, there isn't even a `-c` anywhere in the repo that I could fine.

Seems like there is something strange going on with Gitlab CI.  Have you tried running it locally on your machine?

**Comment by kylemanna** [comment]
Still an issue?

**Comment by locallycompact** [comment]
Yes, looks like it's still an issue with gitlab-ci. I find docker awkward and annoying outside of gitlab so I tend not to use it directly, but I'll give it a try when I can.
